### ISSUE TITLE
Issues using Clio v6.0.2.41+8e74cf2cf2386eaedff32cb3321b745828310567 with Creatio 8.1.1.x

### ISSUE THREAD

**Issue by unusen** [Advance-Technologies-Foundation/clio#259]
Hi Clio Team! How are you? We are experiencing issues using Clio with Creatio 8.1.1.x
We could register the enviroment but for example when we tried to install clio gate we received an error. 
When we tried to modify a system setting (see the image attached) the message result was sucesfully but nothing changed on Creatio side.
![image](https://github.com/Advance-Technologies-Foundation/clio/assets/46381083/44d1e346-fbbd-40d0-9498-4e14ec1d3dc9)

Using Clio Visual Code extension I noticed that all the 8.1.1 sites I've been registered has a red icon instead the green one.
Thank you!
Regards

**Comment by kirillkrylov** [comment]
@unusen 
Good morning, could please tell us more about the error you received when tried installing 'clio-gate'.

The screen you provided for SysSetting seems to exhibit correct behavior.

**Comment by unusen** [comment]
Hi Kyril! How are you?
Below the error trying to install clio gate:
[image: image.png]
Regarding this part of your answer "The screen you provided for SysSetting
seems to exhibit correct behavior":
As you commented the result shows everything is working fine but nothing
changes on Creatio, the system setting has the same value as before.
Regards.


*Uriel Nusenbaum*

Solutions Architect

**Comment by kirillkrylov** [comment]
@unusen 
Its strange, I tried it on 8.1.1 and I cannot seem to reproduce the issue, `clio set-syssetting SchemaNamePrefix BG1 Text -e my-env` works correctly

does `clio open -e ubscloudprod` open correct env ?

![image](https://github.com/Advance-Technologies-Foundation/clio/assets/7285114/2dde58bf-5c50-4845-a9e6-23b3081657f2)

**Comment by unusen** [comment]
Hi Kirill! How are you? As always, I appreciate your help.
My concern was not in the cloud environment but rather on the local site.
With the command you gave me I was able to detect the error. Basically I
had the site registered with localhost, but the IIS binding was with the
name of my computer. I added the binding in IIS and everything is working.

Thank you!
Regards.


*Uriel Nusenbaum*

Solutions Architect
